cfme/automate/explorer/method.py

```diff
@@ -433,7 +434,7 @@ def tree_path(self):
     def create(
             self, name=None, display_name=None, location='inline', script=None, data=None,
             cancel=False, validate=True, repository=None, playbook=None, machine_credential=None,
-            hosts=None, max_ttl=None, escalate_privilege=None, verbosity=None,
+            hosts=None, max_ttl=None, logging_output=None, escalate_privilege=None, verbosity=None,
```

Comment:
This attribute should be on the `Method` entity, and passed to the instantiate call as well.

Reply:
@aljazkosir This is required - the Entity for this collection doesn't have this attribute. It should be stored with the entity object, otherwise its invisible to the framework outside of this create method, where its passed to the form fill.